### PR TITLE
Updated description panel to support proper text wrapping in the title

### PR DESCRIPTION
**Pull request by cre8ivepark**
## Overview
Updated description panel to fix title text overlapping issue. #8502 
- Removed the title text element in the SceneDescriptionPanelRev.prefab
- Instead of using a separate title text element, now it uses TMP's `<size> and <b> tag` to create the title text. This makes the content text properly pushed down when there is a multi-line title.
- Updated all example scene's description text to include this new title text markup. (No content changes in the scenes)

## Changes
- Fixes: #8502 

## Screenshots
2020-09-09 20_22_53-Unity 2018 4 26f1 Personal -  PREVIEW PACKAGES IN USE  - HandInteractionExamples

Now uses markup to create bold and large title text instead of using a separate TMP object.
2020-09-09 20_33_18-Unity 2018 4 26f1 Personal -  PREVIEW PACKAGES IN USE  - ClippingExamples unity
2020-09-09 20_24_28-Unity 2018 4 26f1 Personal -  PREVIEW PACKAGES IN USE  - HandInteractionGestureE

### CODE REVIEW

**Review by MaxWang-MS — APPROVED** [review]
Looks good to me!

File: Assets/MRTK/Examples/Experimental/ExamplesHub/Profiles/MRTKExamplesHubSceneSystemProfile.asset [hunk 1]
```diff
@@ -33,8 +33,8 @@ MonoBehaviour:
   contentScenes:
   - Name: MRTKExamplesHubMainMenu
     Path: Assets/MRTK/Examples/Experimental/ExamplesHub/Scenes/MRTKExamplesHubMainMenu.unity
-    Included: 0
-    BuildIndex: -1
+    Included: 1
```
Comment (keveleigh) [inline Assets/MRTK/Examples/Experimental/ExamplesHub/Profiles/MRTKExamplesHubSceneSystemProfile.asset:36]: I don't think we want to include this. It keeps getting overwritten by the default state of the repo

File: Assets/MRTK/Examples/Demos/StandardShader/Scenes/MaterialGallery.unity [hunk 2]
```diff
@@ -4582,72 +4778,267 @@ Transform:
...
-      value: 0
-      objectReference: {fileID: 0}
-    - target: {fileID: 4085006294760784, guid: 28f445004b874b329a03cd0e3cc63e5d, type: 3}
-      propertyPath: m_LocalPosition.y
-      value: 1
-      objectReference: {fileID: 0}
-    - target: {fileID: 4085006294760784, guid: 28f445004b874b329a03cd0e3cc63e5d, type: 3}
-      propertyPath: m_LocalPosition.z
-      value: 0
+  m_Name: ShaderBallClippingBox (Instance)
```
Comment (keveleigh) [inline Assets/MRTK/Examples/Demos/StandardShader/Scenes/MaterialGallery.unity:4788]: @vaoliva @Cameron-Micka Should these instanced materials be getting serialized into the scene?

Comment (Cameron-Micka) [reply Assets/MRTK/Examples/Demos/StandardShader/Scenes/MaterialGallery.unity:4788]: This was the old (current?) behavior of the MaterialInstance. But, @vaoliva may have addressed this with her latest changes?

Comment (keveleigh) [reply Assets/MRTK/Examples/Demos/StandardShader/Scenes/MaterialGallery.unity:4788]: Yeah, it seems like there are some other scenes that both had and continue to have them serialized. This was the only one that looked like it had new materials serialized, so I wanted to make sure it's expected!

Comment (vaoliva) [reply Assets/MRTK/Examples/Demos/StandardShader/Scenes/MaterialGallery.unity:4788]: @keveleigh thanks for spotting this. I will check a change I have made to solve an issue with text mesh pro.

Comment (keveleigh) [reply Assets/MRTK/Examples/Demos/StandardShader/Scenes/MaterialGallery.unity:4788]: Chatted with @vaoliva offline: I'm gonna go ahead and merge this PR, then we can clean up any existing instances after the change referenced above

Comment (vaoliva) [reply Assets/MRTK/Examples/Demos/StandardShader/Scenes/MaterialGallery.unity:4788]: Opened #8527 to investigate the issue.

**Pull request merged**